thunder/benchmarks/targets.py

```diff
@@ -812,3 +814,29 @@ def test_resnet50(benchmark, executor: Callable, compute_type: ComputeType):
     fn = executor(b.fn())
 
     benchmark_for_compute_type(compute_type, benchmark, fn, args, kwargs)
+
+
+#
+# Thunder dynamo benchmarks
+#
+
+
+def test_dynamo_LlamaMLPBenchmark(benchmark):
+    backend = ThunderCompilerGraphBenchmarking(benchmark)
+
+    bench: Benchmark = LlamaMLPBenchmark(
+        config="Llama-2-7b-hf",
```

Comment:
Q: would there be a plan to use Llama-3-8B by default? I'm just wondering

Reply:
At the moment, I have just randomly chosen this model as an example of how to use the graph-by-graph benchmark.